ably/options.go

```diff
@@ -713,6 +797,10 @@ func WithDial(dial func(protocol string, u *url.URL, timeout time.Duration) (pro
 
 func applyOptionsWithDefaults(os ...ClientOption) *clientOptions {
 	to := defaultOptions
+	// No need to set hosts by default
+	to.RESTHost = ""
+	to.RealtimeHost = ""
+	to.FallbackHosts = nil
```

Comment:
@tcard I have updated this one, to support env. fallbacks functionality, we need a way to detect which hosts were set and which were not, so we can't set them by default, they should be calculated as a part of fallback logic.